README.md

```diff
@@ -1,70 +1,137 @@
+  <br/>
+  <h3><b>Countries Stats</b></h3>
 
-## Available Scripts
+</div>
 
-In the project directory, you can run:
+# 📗 Table of Contents
 
-### `npm start`
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+     - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#triangular_flag_on_post-deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
 
-Runs the app in the development mode.\
-Open [http://localhost:3000](http://localhost:3000) to view it in your browser.
+<!-- PROJECT DESCRIPTION -->
 
-The page will reload when you make changes.\
-You may also see any lint errors in the console.
+# Countries Stats<a name="about-project"></a>
 
-### `npm test`
+It is an application that obtains country data from the API https://restcountries.com where it shows by default 6 countries and you can see the details of that country, you can also search for countries by their name that will automatically change the first letter to capital letters to Filtering in the API database automatically hides the 6 example countries and shows in real time those that match the search with their respective button to view details.
 
-Launches the test runner in the interactive watch mode.\
-See the section about [running tests](https://facebook.github.io/create-react-app/docs/running-tests) for more information.
+You will create an simplified version of it that allows you to:
 
-### `npm run build`
+- Display a list of 6 countries
+- Search country by name
+- See details of a country
+## 🛠 Built With <a name="built-with"></a>
 
-Builds the app for production to the `build` folder.\
-It correctly bundles React in production mode and optimizes the build for the best performance.
+### Tech Stack <a name="tech-stack"></a>
 
-The build is minified and the filenames include the hashes.\
-Your app is ready to be deployed!
+- Languages: HTML, CSS, JavaScript.
+- Technologies used: React, Linters, Github, Visual Code.
 
-See the section about [deployment](https://facebook.github.io/create-react-app/docs/deployment) for more information.
+<!-- Features -->
 
-### `npm run eject`
+### Key Features <a name="key-features"></a>
+- **A responsive webpage to present my knowledge, experience and projects in programming.**
 
-**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+## 🚀 Live Demo <a name="live-demo"></a>
 
-If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+- [Live Demo Link](https://carlosz96.github.io/countries-database/)
```

Comment:
- [ ] Kindly check your live demo  the project not work or display as expected please fix it 
- [ ]  According to the project requirement The project should deployed properly, the home and detail page works as expected.
A link to the deployed project is present in the README.